core/services/relay/evm/chain_writer.go

```diff
@@ -191,7 +191,7 @@ func (w *chainWriter) GetFeeComponents(ctx context.Context) (*commontypes.ChainF
 		return nil, fmt.Errorf("gas estimator not available")
 	}
 
-	fee, _, err := w.ge.GetFee(ctx, nil, 0, w.maxGasPrice, nil)
+	fee, _, err := w.ge.GetFee(ctx, nil, 0, w.maxGasPrice, nil, nil)
```

Comment:
Is it possible for this being nil to cause errors?

Reply:
It shouldn't in this case. This parameter is only used for gas limit estimation and every parameter for that`eth_estimateGas` is optional. Since we're passing nil for `calldata`, `from`, and `to`, the gas estimation call would just return some default value. But that shouldn't matter here since the gas limit isn't being used.